### ISSUE TITLE
Can't set up Panel 

### ISSUE THREAD

**Issue by RyukaMoon** [erikrichardlarson/unbox#73]
Hi, 

I already subscibed to pro to be able to set everything up but 

1. I get the Javascript error on Windows as well, it still displays what is playing on Virtual DJ though
2. The Track list panel stays empty.. no tracks are added
3. How exactly do I intergrate it into OBS? I'm don't know much about these things, please help^^

Thanks a lot,
Hugs
Celina

**Comment by RyukaMoon** [comment]
I mean on Twitch the panel stays empty, and in the unbox program I get the java error but it still displays my tracks there.

**Comment by erikrichardlarson** [comment]
Hey sorry you're running into issues here! 

What does the Javascript error say? 

For the panel, did you add the token to the Twitch config? And did it say that it saved successfully? Also feel free to link your Twitch here and I can take a look at your Twitch profile.

For OBS, you can copy the URL on step 3 of the unbox app, and paste that URL into a browser source in OBS.

**Comment by RyukaMoon** [comment]
![javaerror](https://user-images.githubusercontent.com/96054566/150019602-4d3ae1f5-d680-4207-a752-ead49688a3a9.PNG)

**Comment by RyukaMoon** [comment]
this is the error I get :( Thanks for your help

**Comment by RyukaMoon** [comment]
I fixed the OBS thing :)

**Comment by erikrichardlarson** [comment]
Nice, glad OBS is working, so it looks like the error is caused by an undefined Artist, is it possible you're playing a track with an empty Artist field? Also is the Twitch panel working now?

**Comment by RyukaMoon** [comment]
Well the Java error pops up at the very second I open unbox :(
Then about 3 songs are displayed with a delay in my OBS with constant Java error messages (always the same one from the screenshot) and then it stops working completely. would be sooo great if we could fix this!

**Comment by RyukaMoon** [comment]
yours is the only application I found that can be connected with Virtual DJ so I would be really glad if we can fix it

**Comment by RyukaMoon** [comment]
![Unbenannt](https://user-images.githubusercontent.com/96054566/150207124-6d20a25a-1095-4dbc-8a90-a77b91df39ca.PNG)

**Comment by RyukaMoon** [comment]
Also this config file looks like this, is that right? as java seems to have an issue with this json file

**Comment by erikrichardlarson** [comment]
Hmm the config should update so it shouldn't be an empty json file like that, what is the Javascript error you're seeing now? I think it's different from the screenshot above.

**Comment by erikrichardlarson** [comment]
Also can you confirm that the Artist field isn't empty for the song you're playing? Feel free to send over your `VirtualDJ\History\tracklist.txt` file so we can inspect it a bit.

**Comment by RyukaMoon** [comment]
[tracklist.txt](https://github.com/erikrichardlarson/unbox/files/7906700/tracklist.txt)

Here is my tracklist file, thanks for the help

**Comment by RyukaMoon** [comment]
![Unbenannt](https://user-images.githubusercontent.com/96054566/150375594-33b52704-a9ab-4242-b74f-46c4ab155bd6.PNG)

its always this error with this message

**Comment by erikrichardlarson** [comment]
Ah yeah it looks like some of your tracks have an empty Artist field, so you'll want to update those tracks to contain an Artist value. That will take care of the first Javascript error you're seeing. And for the Javascript error here related to the `config.json` file, it's just a permissions error, if you run unbox as admin / move to your desktop then you should be good to go. Let me know if you still run into issues.

**Comment by RyukaMoon** [comment]
For the config file Java error : Virtual DJ is on my main hard drive and unbox on my dekstop and even if I run it as admin the error continues

**Comment by RyukaMoon** [comment]
well maybe it just doesnt work on my laptop.... thanks for trying to help me though! :)

**Comment by erikrichardlarson** [comment]
Yeah no prob, hmm can you try this: Right click the `unbox_windows` folder, then go to unbox_windows > Properties > Security Tab > Click Edit > Select / give Full Control to your user.

**Comment by RyukaMoon** [comment]
Tried, it still has the trouble with the config. json file :( so sad

**Comment by RyukaMoon** [comment]
can you like give me this file filled with the stuff it should update itself with so that its not empty? Would that work?

**Comment by erikrichardlarson** [comment]
The writes will happen on an ongoing basis so you'll want the config to update as expected. Can you try excluding the `unbox_windows` folder from Windows Defender: https://support.microsoft.com/en-us/windows/add-an-exclusion-to-windows-security-811816c0-4dfd-af4a-47e4-c301afe13b26
